guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -21,6 +21,15 @@ ifdef::client-content-dnf[]
 ----
 # {client-package-install-el8} katello-pull-transport-migrate
 ----
+ifndef::satellite[]
++
+If you have EPEL enabled on your host, ensure that your host installs `yggdrasil` from {project-client-name}:
++
+*** Lock the version by using `echo "yggdrasil-_My_Package_Version_" >> /etc/dnf/plugins/versionlock.list`.
```

Comment:
Have you considered including the command in a code block as usual? Seeing it inline like this is a bit weird.

Reply:
I did not consider this up until now. Applied.